CHANGELOG.md

```diff
@@ -4,6 +4,30 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 7.89.0
```

Comment:
Let's call out the `configureScope`, `pushScope` and `popScope` deprecations maybe?